domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewPersistenceAdapter.kt

```diff
@@ -12,7 +12,7 @@ interface ReviewPersistenceAdapter {
   fun getCompletedReviews(prisonNumber: String): List<CompletedReview>
 
   /**
-   * Creates and returns a new [CompletedReview] using the specified [CreateCompletedReviewDto] and deadlineDate.
+   * Creates and returns a new [CompletedReview] using the specified [CreateCompletedReviewDto] and [ReviewSchedule].
    */
-  fun createCompletedReview(createCompletedReviewDto: CreateCompletedReviewDto, deadlineDate: LocalDate): CompletedReview
+  fun createCompletedReview(createCompletedReviewDto: CreateCompletedReviewDto, reviewSchedule: ReviewSchedule): CompletedReview
```

Comment:
The impl now needs the ReviewSchedule when creating a Completed Review, because we now need the reference of the schedule (as well as the deadline date, which we can get from the review schedule)